src/samples/DistanceImage.hpp

```diff
@@ -79,16 +79,15 @@ namespace samples
 	template <class Scalar_>
 	bool getScenePoint( size_t x, size_t y, Eigen::Matrix<Scalar_,3,1>& point ) const
 	{
-	    point = Eigen::Matrix<Scalar_,3,1>( (x*scale_x)+center_x, (y*scale_y)+center_y, 1.0 );
```

Comment:
> I actually did miss the tests ... need more sleep or less work maybe...

No you didn't, I used the incorrect tense.

> I don't believe that you verify that the point arg is left as-is if the coordinates are wrong.

I didn't check this on purpose, because I wasn't sure if we want to enforce it.
The unit tests verify now that the args are not modified and I fixed it in getImagePoint as well.